dist/_index.scss

```diff
@@ -1,7 +1,6 @@
 @import "./_variables.scss";
 @import "./_animation.scss";
 @import "./_responsive.scss";
-@import "./_spacing.scss";
```

Comment:
I decided to remove the existing spacing mixins, as I'm under the impression they won't be needed with the new scale.